### PR TITLE
Improved helper scripts to be more restrictive.

### PR DESCRIPTION
**Pull request by utoni**

### CODE REVIEW

File: utils/whatsapp_ip_addresses_download.sh
Comment (IvanNardi) [inline utils/whatsapp_ip_addresses_download.sh:1]: could you check the WA change?
It seems that fetching this list now fails, while before it was ok, IMO

Comment (utoni) [reply utils/whatsapp_ip_addresses_download.sh:1]: On my local machine, the old URL was failing. I thought that Facebook may give different results depending on the UA/Referer, but that seems not to be the case here.

Comment (IvanNardi) [reply utils/whatsapp_ip_addresses_download.sh:1]: Strange...

Comment (utoni) [reply utils/whatsapp_ip_addresses_download.sh:1]: Reverted to the old URL/Regex.

**Review by IvanNardi — APPROVED** [review]

**Pull request merged**